src/snowflake/snowpark/dataframe.py

```diff
@@ -1246,6 +1262,19 @@ def sort(
             return self._with_plan(self._select_statement.sort(sort_exprs))
         return self._with_plan(Sort(sort_exprs, self._plan))
 
+    def alias(self, name: str):
```

Comment:
As talked offline, pls mark it as `@experimental`.
@sfc-gh-mrojas Is it good for you? I'm afraid this may get user feedback and have breaking changes in the future so would like to mark it experimental first.

Reply:
@sfc-gh-yixie I am not sure if it makes sense to have an experimental feature. I think that in that case I would prefer to not proceed and introduce automation in our conversion tools